packages/blocks/src/api/registration.js

```diff
@@ -245,6 +249,11 @@ export function registerBlockType( name, settings ) {
 		return;
 	}
 
+	settings.attributes = addImplicitAttributes(
```

Comment:
I'd call these `supportedAttributes` maybe

Reply:
I like that.